roles/olm_operator/tasks/main.yml

```
@@ -55,10 +55,23 @@
     - operator_packagemanifest is defined
     - operator_packagemanifest.resources is defined
     - operator_packagemanifest.resources | length
-  failed_when: operator_packagemanifest is skipped
+
+- name: Set package as not found
+  ansible.builtin.set_fact:
+    olm_operator_found: false
+  when: operator_packagemanifest is skipped
     or operator_packagemanifest.resources is undefined
     or operator_packagemanifest.resources | length != 1
 
+- name: Fail if not found and nor skippable
+  ansible.builtin.fail:
+    msg: >
+      The operator {{ operator }} is not found in the catalog
+      and skippable is {{ olm_operator_skippable }}.
+  when:
+    - olm_operator_skippable is false
```

Beto, here shouldn't be something like this:
```suggestion
    - not olm_operator_skippable | bool
```
I haven't used an expression like you did, I wonder if it would be treated as a string and not as a boolean?